src/stores/connectors/secrets/get.go

```diff
@@ -11,6 +12,14 @@ import (
 func (c Connector) Get(ctx context.Context, id, version string) (*entities.Secret, error) {
 	logger := c.logger.With("id", id, "version", version)
 
+	if version == "" {
+		var err error
+		version, err = c.db.GetLatestVersion(ctx, id, false)
```

Comment:
forgot to check that the error is not nil ? Surely missing some unit test to check this